.mvn/extensions.xml

```diff
@@ -0,0 +1,7 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
```

Comment:
no need of incremental here. not sure what is the added value here,

Reply:
What is the reason for requesting changes? The presence of incrementals support?

Reply:
what is the added value for this project?

Reply:
As stated in the PR description, a slimmer plugin `.hpi` (one that doesn't bundle Guava and Guice).

Reply:
I do not say anything about the rest of the PR but incremental as it's not in the description.

Reply:
What is the issue with incrementalizing this plugin?

Reply:
My question is why doing it what is the added value :)

Reply:
it's not a really used plugin, And this change is not part of the PR description and has nothing to do with the rest of the PR

Reply:
[JEP-305](https://github.com/jenkinsci/jep/tree/master/jep/305) explains the motivation. Notably, having incrementals automatically published makes it easy to produce builds for users to test, as in [JENKINS-67163](https://issues.jenkins.io/browse/JENKINS-67163).

Reply:
> [JEP-305](https://github.com/jenkinsci/jep/tree/master/jep/305) explains the motivation. Notably, having incrementals automatically published makes it easy to produce builds for users to test, as in [JENKINS-67163](https://issues.jenkins.io/browse/JENKINS-67163).

frankly I don't think we need such complexity here.. not sure we have users asking for a such builds.

Anyway my point is this PR mix up too many unrelated things!